### PR TITLE
PHPLIB-930: Update expected error in fle2-InsertFind-Unindexed test

### PR DESCRIPTION
**Pull request by levon80999**
https://jira.mongodb.org/browse/PHPLIB-930
Synced with mongodb/specifications@1a911b5e8911f6f57426ec1d451dbdc19480afd8

### CODE REVIEW

File: tests/SpecTests/client-side-encryption/tests/fle2-InsertFind-Unindexed.json [hunk 1]
```diff
@@ -241,10 +241,10 @@
...
           "result": {
-            "errorContains": "Cannot query"
+            "errorContains": "encrypt"
           }
         }
       ]
     }
   ]
-}
+}
```
Comment (jmikola) [inline tests/SpecTests/client-side-encryption/tests/fle2-InsertFind-Unindexed.json:250]: The original file in [mongodb/specifications@1a911b5](https://github.com/mongodb/specifications/blob/1a911b5e8911f6f57426ec1d451dbdc19480afd8/source/client-side-encryption/tests/legacy/fle2-InsertFind-Unindexed.json) includes a trailing new line, which this PR is removing. Please update the file.

If this was inadvertently caused by your IDE trimming the trailing newline automatically, you may need to adjust that behavior.

**Comment by jmikola** [comment]
Please remove "https://jira.mongodb.org/browse/PHPLIB-930" from the commit message. I only drop that in the PR descriptions because GitHub sometimes fails to render the [configured autolinking](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources). The commit itself only needs the issue number as a prefix, though.

And friendly reminder to bump the status of [PHPLIB-930](https://jira.mongodb.org/browse/PHPLIB-930) after opening a PR.

File: tests/SpecTests/client-side-encryption/tests/fle2-InsertFind-Unindexed.json (hunk 1 above)
Comment (levon80999) [reply tests/SpecTests/client-side-encryption/tests/fle2-InsertFind-Unindexed.json]: Done

**Comment by jmikola** [comment]
Diff looks fine now, but I see the following is now missing from the commit:

```
Synced with mongodb/specifications@1a911b5e8911f6f57426ec1d451dbdc19480afd8
```

Please make sure that ends up in the final commit when merging.

**Review by jmikola — APPROVED** [review]
Diff looks fine now, but I see the following is now missing from the commit:

```
Synced with mongodb/specifications@1a911b5e8911f6f57426ec1d451dbdc19480afd8
```

Please make sure that ends up in the final commit when merging.

**Pull request merged**